app/models/user_mailbox.rb

```diff
@@ -0,0 +1,43 @@
+class UserMailbox
+  attr_reader :user
+
+  def initialize(user)
+    @user = user
+  end
+
+  def inbox
+    messages = user.mailbox.inbox
+    messages.each{|m| m.mark_as_read(user)}
+  end
+
+  def delete_all
+    user.mailbox.inbox.each do |msg|
+      delete_message(msg)
+    end
+    empty_trash(user)
+  end
+
+  def destroy message_id
```

Comment:
Always surround params with parentheses in method definitions. (Parroting Rubocop.)